data/ecosystems/o/osmosis.toml

```diff
@@ -218,3 +252,30 @@ url = "https://github.com/osmosis-labs/wasmd"
 
 [[repo]]
 url = "https://github.com/osmosis-labs/wormhole"
+
+[[repo]]
+url = "https://github.com/forbole/big-dipper-2.0-cosmos"
```

Comment:
Big Dipper is more of a general purpose block explorer for Cosmos.  Does it belong in Cosmos ecosystem rather than a direct repo in Osmosis?

Reply:
Can be removed.  I guess I don't see much difference between Osmosis and the Cosmos ecosystem at large 😉

Reply:
The Ethereum.toml contains generalized cross-chain products too such as metamask?